### DISCUSSION TITLE
now 403 with "Invalid JWT Token" on TI RU

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#60] [Q&A]
Requests habe been working this morning using TI RU.
Now we get a 403 with body "Invalid JWT Token" while using API for retrieving digital certificate.
Some changes or work/install at progress?

**Comment by oliver-steinbrecher** [comment]
Please retry

**Reply by mge-medisoftware** [reply]
yes, thanks, works again since yesterday in TI RU. Now also with "Genesenenzertifikat"!

**Reply by fue-medatixx** [reply]
Which BSNR do you use for the test? In our attempts, the error with the whitelist still comes up in RU

**Reply by dirkexe** [reply]
e.g. 191872682

**Reply by fue-medatixx** [reply]
The RU is "greener" on your side :-) We get
Error: 0 : 09.06. 14:05:44,617: Certificate request error: 403: Forbidden: Sorry, the credential is invalid:Id not whitelisted -> id:191872682 cert:149829843956210962692896088857717103075125879956

But IBM will have a look.

**Reply by nicolaslang1** [reply]
Has this been fixed? I got the error with the "the credential ist invalid:Id not whitelisted" when i forgot to add the BearerAuthToken. After that i receive the Invalid JWT Token too.
